### PR TITLE
Fix multi mspec with developing ruby

### PR DESCRIPTION
**Pull request by nobu**
When any commits haven't been pushed into the main repository,
`ruby -v` prints `last_commit=` line after usual description.
Also, when configured to use jemalloc, prints `malloc_conf=` line
too.

`multi_exec` assumes `ruby -v` to print the description only in
one line, it fails in the above cases.

This fixes the failure by printing an empty line at the beginning
in worker processes and the master reads them in paragraph mode.

### CODE REVIEW

**Comment by eregon** [comment]
Is `RUBY_DESCRIPTION` affected by this extra output?
I was thinking to just `print RUBY_DESCRIPTION` in mspec-run to avoid this issue.

**Comment by eregon** [comment]
@nobu 614fd74 should fix it, could you confirm?
I tried locally with ruby-trunk but I don't have the extra lines (maybe because I use git?).

File: lib/mspec/commands/mspec.rb [hunk 1]
```diff
@@ -107,7 +107,7 @@ def multi_exec(argv)
       IO.popen([env, *command], "rb+")
     }
 
-    puts children.map { |child| child.gets }.uniq
+    puts children.map { |child| child.gets("").chomp }.uniq
```
Comment (eregon) [inline lib/mspec/commands/mspec.rb]: The idea is nice, I might use it later or to be more resilient to the interpreter extra output, thanks.

**Comment by nobu** [comment]
Yes, it works now, thank you.

You need a local commit on `origin/trunk`, try `git commit -m TEST --allow-empty`.
